### PR TITLE
Add screenshot tests for most examples in wasm

### PR DESCRIPTION
**Pull request by janpaul123**
screenshots

See individual commits!

Not all screenshots work yet, so I added some TODOs. Makes progress
towards https://github.com/Zaplib/zaplib/issues/29.

Samsung Galaxy has some issues, so I excluded it for the examples, and
added a note in https://github.com/Zaplib/zaplib/issues/67.

### CODE REVIEW

File: zaplib/scripts/ci/browser_tests.sh [hunk 1]
```diff
@@ -5,8 +5,29 @@ set -euxo pipefail
 # Per https://stackoverflow.com/a/16349776; go to repo root
 cd "${0%/*}/../../.."
 
-# Build test_suite.wasm
+# Build the wasm packages that we need in tests.
```
Comment (pankdm) [inline zaplib/scripts/ci/browser_tests.sh:8]: Should we just build everything here for simplicity? E.g. `cargo zaplib build workspace` or something.

Comment (janpaul123) [reply zaplib/scripts/ci/browser_tests.sh:8]: That's what I did first, but this has already become our longest build now, so I figured it's worth it to be specific in order to reduce build times a bit.

Comment (pankdm) [reply zaplib/scripts/ci/browser_tests.sh:8]: Maybe add a simple comment that this is to save on build times so other people coming through this code didn't try to "simplify" it?

Comment (janpaul123) [reply zaplib/scripts/ci/browser_tests.sh:8]: Great idea, will do!

**Pull request merged**